### DISCUSSION TITLE
submit error-----------------

### DISCUSSION THREAD

**Discussion by RobinHan24** [PaddlePaddle/PaddleSpatial#82] [BaiduKDDCup2022]
my paddle version is 2.3.0, but my submit result is Error Occurred. Could you please help me. Thanks a lot.

**Comment by ramber1836** [comment]
Hi, there,

You can provide your team name and the administrator will reply to you in a
couple of days (the workload is hard) by checking the online test log.
Thanks for your understanding.

Regards,
Xinjiang Lu
